Momo's December updates (#836)
akeneo/pim-helpcenter: content/updates/2020-12/3-Export-product-with-assets.md

---
pim_announcement_audience:
- EE
---

# Exports your products with their assets
::: meta-data type="New" features="Collaboration, Productivity" available="late December" link-to-doc="../articles/TODO"

> Collaborating around product data is now more efficient as it's now possible to export products and their associated assets from within the product grid or from your product export profiles. It will save you time whenever you need to share product information with your internal and external teams.

Reviewer: ```suggestion
Collaboration around product data is now more efficient as it's now possible to export products and their associated assets from the product grid or your product export profiles. It will save you time whenever you need to share product information with your internal and external teams.
```

---
pim_announcement_img: ./img/quick-export-products-with-assets.png
pim_announcement_alt_img: "The new asset import profile"
pim_announcement_audience:
- EE
---

# Exports your products with their assets
::: meta-data type="New" features="Collaboration, Productivity" available="late December" link-to-doc="../articles/exports.html#update-an-export-profile"

Collaboration around product data is now more efficient as it's now possible to export products and their associated assets from the product grid or your product export profiles. It will save you time whenever you need to share product information with your internal and external teams.

![Quick export products with their assets](../img/quick-export-products-with-assets.png)

> _How does it work?_ Once the export is completed, you receive an archive containing all the media linked to your products. In the downloaded Excel or CSV file, a new `file path` column allows you to know which assets are linked to which product.

Author: ```suggestion
```

> _How does it works?_ Once the export is completed, you receive an archive containing all the media linked to your products. In the downloaded Excel or CSV file, a new `file path` column allows you to know which assets are linked to which product.

Reviewer 2: ```suggestion
_How does it work?_ Once the export is completed, you receive an archive containing all the media linked to your products. In the downloaded Excel or CSV file, a new `file path` column allows you to know which assets are linked to which product.
```

> _How does it works?_ Once the export is completed, you receive an archive containing all the media linked to your products. In the downloaded Excel or CSV file, a new `file path` column allows you to know which assets are linked to which product.

Reviewer 2: Are you sure you want to explain how it works here? The link to the documentation is not enough? I find it a bit "dangerous" because if some media are "media links", the behavior isn't the same.

Author: Laure, a thought on that? It can indeed be confusing for our customers.
